content/release-10-16-0/streaming-analytics-10-16-0-bundle/10_16_0_6.md

```diff
@@ -0,0 +1,43 @@
+---
+weight: 34
+title: Release 10.16.0.6
+layout: redirect
+---
+
+This release upgrades the Apama-ctrl microservice to use Apama 10.15.4.1 (which is the same as Apama 10.15 Fix 17).
+
+### Fixes
+
+<table>
+<colgroup>
+    <col style="width: 15%;">
+    <col style="width: 70%;">
+    <col style="width: 15%;">
+</colgroup>
+<thead>
+<tr>
+<th style="text-align:left">Component</th>
+<th style="text-align:left">Description</th>
+<th style="text-align:left">Issue</th>
+</tr>
+</thead>
+<tbody>
+
+<tr>
+<td style="text-align:left">Apama-ctrl microservice</td>
+<td style="text-align:left">For reasons of security and performance, the REST endpoint <code>/service/cep/health</code> no longer returns a comprehensive list of status values. All of the same information is still available from REST endpoints under <code>/service/cep/diagnostics/...</code>.</td>
+<td style="text-align:left">PAB-4361</td>
+</tr>
+<tr>
+<td style="text-align:left">Streaming Analytics application</td>
+<td style="text-align:left">To fix a security vulnerability, the Microservice SDK has been updated to version 1016.0.440 to use version 1.2.13 of logback-core and logback-classic.</td>
```

Comment:
Didn't we say that we won't include security related topics in change log ? remove this ?

Reply:
This is not a change log. It's a release note entry in the old style. So I was wondering if we still want to mention this in this case .... ???

Reply:
lets remove it. we need to be sensitive about security issues :D

Reply:
ok, removed.